src/libs/reportUtils.js

```diff
@@ -35,6 +36,20 @@ function sortReportsByLastVisited(reports) {
         .value();
 }
 
+/**
+ * Check whether user can edit report action
+ *
+ * @param {Object} reportAction
+ * @param {String} sessionEmail
+ * @returns {Boolean}
+ */
+function canEditReportAction(reportAction, sessionEmail) {
+    return reportAction.actorEmail === sessionEmail
+        && reportAction.reportActionID
+        && reportAction.actionName !== 'IOU'
```

Comment:
```suggestion
        && reportAction.actionName === 'ADDCOMMENT'
```

Also, you could optionally add a const to CONST.js called `REPORT_ACTION.ACTION_TYPES` so that we're not using a string literal here.